src/Main.hs

```diff
@@ -169,10 +173,12 @@ alex cli file basename script = do
 
    scheme <- getScheme directives
 
+   let o_file_tmp = o_file ++ ".tmp"
```

Comment:
Should this maybe more robust by using a proper temp file?
See e.g. https://hackage.haskell.org/package/base-4.3.0.0/docs/System-IO.html#v:openTempFile

Reply:
I made two changes:

1. Use `openTempFile`
2. Append a `~` to the name of the temporary file

The rationale for (2) is so that the file is treated the same way as backup files by file watching utilities and the like.